_posts/jared_baker/2017-10-20-Day-1-Collaboratory.md

```diff
@@ -0,0 +1,276 @@
+
+<figure>
+    <img id="instance-networks" src="{{site.urlimg}}jared_baker/day1collab/3-instance-networks.png" data-featherlight="#instance-networks"/>
+</figure>
+
+Security groups the firewall for your instance. They define what can and cant go to/from your Instance over the network. Choose the default security group for now as we will modify this later on.
+
+<figure>
+    <img id="instance-secgroups" src="{{site.urlimg}}jared_baker/day1collab/3-instance-secgroups.png" data-featherlight="#instance-secgroups"/>
+</figure>
+
+Add your key pair to the instance. This will ensure that your instance will be injected with the appropriate public key allowing you to authorize yourself against the instance via SSH. At this point we can skip the rest of the Launch Instance wizard and click the 'Launch Instance' button!
+
+<figure>
+    <img id="instance-keypair" src="{{site.urlimg}}jared_baker/day1collab/3-instance-keypair.png" data-featherlight="#instance-keypair" />
+</figure>
+
+<a id="4"></a>
+## Step 4: Assign a floating IP
+In order to reach your instance from the Internet, we need to associate the instance with a Floating IP (aka Public IP). Follow the screenshots!
+
+<figure>
+    <img id="float-associate" src="{{site.urlimg}}jared_baker/day1collab/4-float-associate.png" data-featherlight="#float-associate"/>
+</figure>
+
+<figure>
+    <img id="float-add" src="{{site.urlimg}}jared_baker/day1collab/4-float-add.png" data-featherlight="#float-add" />
+</figure>
+
+<figure>
+    <img id="float-allocate" src="{{site.urlimg}}jared_baker/day1collab/4-float-allocate.png" data-featherlight="#float-allocate" />
+</figure>
+
+<figure>
+    <img id="float-commit" src="{{site.urlimg}}jared_baker/day1collab/4-float-commit.png" data-featherlight="#float-commit" />
+</figure>
+
+<a id="5"></a>
+## Step 5: Allow SSH access to the instance
+Now we have an instance with a Floating IP. We are almost ready to SSH to it! First we will need to modify the security group associated with our instance 'default' to allow SSH access from the Internet.
+
+From the Access & Security, Security Groups tab, click Manage rules
+
+<figure>
+    <img id="secgroup-manage" src="{{site.urlimg}}jared_baker/day1collab/5-secgroup-manage.png" data-featherlight="#secgroup-manage" />
+</figure>
+
+Add a rule for SSH from the drop down. Provide a range of IP addresses to allow SSH FROM, such as your corporate IP space or alternatively you can leave it open to all (0.0.0.0/0 - less secure)
+
+<figure>
+    <img id="secgroup-add" src="{{site.urlimg}}jared_baker/day1collab/5-secgroup-add.png" data-featherlight="#secgroup-add" />
+</figure>
+
+<a id="6"></a>
+## Step 6: SSH to the instance
+It's time to SSH to your instance.
+
+You will need:
+- The floating IP address of the instance
+- Your SSH private key (the .pem downloaded in step 2)
+- The username, which is standardized based on what OS/Image used (ubuntu, centos, debian)
```

Comment:
Maybe add a note and a screenshot about the username for each base image being provided in the image description?